spec/classes/server_spec.rb

```diff
@@ -31,7 +31,7 @@
             :servertype   => server_type,
             :storeconfigs => 'puppetdb',
             :environmentpath => '$confdir/environments',
-            :basemodulepath => '/etc/puppet/modules:/usr/share/puppet/modules',
+            :basemodulepath => ['/etc/puppet/modules', '/usr/share/puppet/modules'],
```

Comment:
ah, yes, absolutely. There's not very much stuff breaking in puppet 4. @tampakrap for simple future parser compatibility, if the code is covered by rspec-puppet tests in this repo then it's already getting tested on the travis-ci tests. For other code, I'd just run puppet --parser future to test.